Sources/PackageDescription/PackageRequirement.swift

```diff
@@ -192,10 +205,12 @@ extension Package.Dependency {
     /// versions, and may require you to modify your code when they update.
     /// The version rule requires Swift packages to conform to semantic
     /// versioning. To learn more about the semantic versioning standard,
-    /// visit [semver.org](https://semver.org).
+    /// visit the [Semantic Versioning 2.0.0](https://semver.org) website.
```

Comment:
I think it's better to either leave it as it currently is, or update the URL to `https://semver.org/spec/v2.0.0.html`

Reply:
I went with the second option, and I changed it in other places in `PackageDescription` also.